daemon/options.c

```diff
@@ -231,6 +241,10 @@ static void config_register_opts(struct lightningd_state *dstate)
 	opt_register_arg("--ipaddr", opt_set_ipaddr, NULL,
 			   &dstate->config.ipaddr,
 			   "Set the IP address (v4 or v6) to announce to the network for incoming connections");
+
+	opt_register_arg(
+	    "--network", opt_set_network, NULL, ld_from_dstate(dstate),
+	    "Select the network parameters (bitcoin, testnet, regtest, or litecoin, default: testnet)");
```

Comment:
Maybe implement opt_show_network (instead of handing NULL here), which means opt will actually show the default setting.  Hmm,  but that means --help will segv on the legacy daemon I think.  So just put a fixme here, maybe.